layer1/bridge/consensus/assert.mligo

```diff
@@ -0,0 +1,4 @@
+
+let assert_msg ((message, condition): (string * bool)) =
```

Comment:
remove this module and just use `assert_with_error` function provided by LIGO
https://ligolang.org/docs/language-basics/exceptions/#assertions
WARNING paramter are reverted: 
`val assert_with_error (bool * string) -> unit`